spec/dummy/config/initializers/assets.rb

```diff
@@ -1,5 +1,13 @@
 # frozen_string_literal: true
 
-if Gem::Version.new(Sprockets::VERSION) < Gem::Version.new('4')
-  Rails.application.config.assets.precompile += %w[manifest.js]
+Rails.application.config.assets.precompile +=
+  if Gem::Version.new(Sprockets::VERSION) < Gem::Version.new('4')
+    %w[manifest.js]
+  else
+    %w[day.css night.css email.css]
```

Comment:
this shouln't be necessary, the manifest file is already included https://github.com/rails/sprockets-rails/blob/master/lib/sprockets/railtie.rb#L107

Reply:
Done